### ISSUE TITLE
API design: only one argument (eject only one dependency at a time)

### ISSUE THREAD

**Issue by brillout** [snake-py/eject#2]
I think we'll make our lives easier, as well as having a simpler interface for the user, if we eject only one package at a time. In other words: 

```diff
- cli.command('[...dependencies]', 'Ejects a dependency from node_modules', {
+ cli.command('<dependency>', 'Eject dependency from node_modules', {
```

**Comment by snake-py** [comment]
You are right this would simplify the script, but why now remove the feature if it is already present?

**Comment by snake-py** [comment]
The longer I think about the more it makes sense.

**Comment by brillout** [comment]
Yea I really think it's better. In the vast majority of cases, users will eject only one package. The user can re-run `eject` if he wants to eject multiple packages anyways. I think it adds more confusion than anything.

Strictly speaking it's a breaking change (so in theory we should publish it as `2.0.0`) but, since no one is using `eject`, we can safely "cheat" and release it as `1.0.1`.
